src/main/java/org/skriptlang/skript/bukkit/damagesource/elements/ExprCausingEntity.java

```diff
@@ -37,8 +38,11 @@ set the damage location to location(0, 0, 10)
 @RequiredPlugins("Minecraft 1.20.4+")
 public class ExprCausingEntity extends SimplePropertyExpression<DamageSource, Entity> implements DamageSourceExperimentSyntax {
 
-	static {
-		registerDefault(ExprCausingEntity.class, Entity.class, "(causing|responsible) entity", "damagesources");
```

Comment:
[Simple]PropertyExpression has a method for constructing the info for you. Use that instead, but you can still modify the returned builder to set a supplier.